source/spawn/spawn-processes.ads

```diff
@@ -225,6 +233,8 @@ private
    subtype Pipe_Kinds is Internal.Pipe_Kinds;
    subtype Standard_Pipe is Pipe_Kinds range Stdin .. Stderr;
 
+   type Pipe_Flags is array (Standard_Pipe) of Boolean;
```

Comment:
Stderr never uses PTY, right? Shall we exclude it from Pipe_Flags indexes?

Reply:
I see, it is used in the last commit.

Reply:
same